app/src/main/java/org/simple/clinic/home/overdue/appointmentreminder/AppointmentReminderSheet.kt

```diff
@@ -110,6 +111,19 @@ class AppointmentReminderSheet : BottomSheetActivity() {
     decrementDateButton.isEnabled = state
   }
 
+  private fun textForAppointmentReminder(timeAmount: Int, chronoUnit: ChronoUnit): String {
+    val quantityStringResourceId = when (chronoUnit) {
+      ChronoUnit.DAYS -> {
+        R.plurals.appointmentreminder_days
+      }
+      ChronoUnit.WEEKS -> {
+        R.plurals.appointmentreminder_weeks
+      }
+      else -> throw AssertionError()
```

Comment:
Assertion error is meant to be thrown in tests. Can you instead use something that exposes the actual problem and add a message?

Ex:
```kotlin
else -> throw IllegalArgumentException("$chronoUnit is unsupported type for appointment reminders")
```

Reply:
Done